tests/pytorch/test_pytorch_model_export.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import print_function
+
+import unittest
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+
+import mlflow.pytorch
+from mlflow import pyfunc
+from mlflow import tracking
+from mlflow.utils.file_utils import TempDir
+
+
+class Flatten(nn.Module):
+    def forward(self, x):
+        return x.view(x.shape[0], -1)
+
+
+class TestModelExport(unittest.TestCase):
```

Comment:
We're currently working on moving our testing suites to [pytest](https://docs.pytest.org/en/latest/). Do you think you could refactor your tests to make use of pytest? You can see an example at [test_databricks.py](https://docs.pytest.org/en/latest/).

Reply:
@juntai-zheng yes, I can refactor that